articles/cosmos-db/create-graph-java.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create an Azure Cosmos DB graph database with Java
+title: 'Quickstart: Build an graph database in Azure with Java'
```

Comment:
We usually like the service name in the title - "Azure Cosmos DB"
"an" is incorrect grammar, instead use  "a"